docker/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:19.10
+
+RUN apt update \
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/manual/dweindl/pyPESTO/pullRequest?prid=5388382)